pkg/controller/controller.go

```diff
@@ -247,12 +251,17 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 	if err != nil {
 		return nil, err
 	}
-	// create random share name
-	share := fmt.Sprintf("%s-%s", p.volumeNamePrefix, strings.Replace(string(uuid.NewUUID()), "-", "", -1)[0:p.volumeNameUUIDLength])
+	// create persistent name based on a volumeNamePrefix and volumeNameUUIDLength
```

Comment:
nit: put volume name generation code in to a new method for clarity.

Reply:
Do you mean make this code into a separate func?

Reply:
yes, separate function please

Reply:
done